sky/clouds/service_catalog/__init__.py

```diff
@@ -46,7 +46,10 @@ def _map_clouds_catalog(clouds: CloudFilter, method_name: str, *args, **kwargs):
             raise AttributeError(
                 f'Module "{cloud}_catalog" does not '
                 f'implement the "{method_name}" method') from None
-        results.append(method(*args, **kwargs))
+        return method(*args, **kwargs)
+
+    results = subprocess_utils.run_in_parallel(_execute_catalog_method,
+                                               list(clouds), len(clouds))
```

Comment:
style nit: use kwargs for non-obvious argument meaning : )

Reply:
Thanks!